### PR TITLE
Fix/pdosviewer

### PR DESCRIPTION
**Pull request by cpignedoli**
Fix a bug where the viewer was crashing when loading old pdos workchains

### CODE REVIEW

File: surfaces_tools/widgets/pdos.py [hunk 1]
```diff
@@ -40,13 +40,16 @@ def read_and_process_pdos_file(pdos_path):
     return out_data, kind
 
 
-def process_pdos_files(scf_calc, newversion=True):
+def process_pdos_files(pdos_wk, newversion=True):
```
Comment (yakutovicha) [inline surfaces_tools/widgets/pdos.py:43]:
```suggestion
def process_pdos_files(pdos_workchain, newversion=True):
```
Please rename all the other occurrences as well.

File: surfaces_tools/widgets/pdos.py [hunk 2]
```diff
@@ -40,13 +40,16 @@ def read_and_process_pdos_file(pdos_path):
...
     if newversion:
-        retr_files = scf_calc.outputs.slab_retrieved.list_object_names()
-        retr_folder = scf_calc.outputs.slab_retrieved
+        retr_files = pdos_wk.outputs.slab_retrieved.list_object_names()
+        retr_folder = pdos_wk.outputs.slab_retrieved
     else:
-        retr_files = scf_calc.outputs.retrieved.list_object_names()
-        retr_folder = scf_calc.outputs.retrieved
+        for wk in pdos_wk.called_descendants:
```
Comment (yakutovicha) [inline surfaces_tools/widgets/pdos.py:48]:
```suggestion
        for process in pdos_wk.called_descendants:
```
Also the other occurrences as well.

**Review by yakutovicha — CHANGES_REQUESTED** [review]
Thanks @cpignedoli. I have a few minor requests.

File: surfaces_tools/widgets/pdos.py (hunk 2 above)
Comment (cpignedoli) [reply surfaces_tools/widgets/pdos.py]: done

**Pull request merged**